config/samples/cxx_example_loadtest_with_pre_built_workers.yaml

```
@@ -0,0 +1,123 @@
+apiVersion: e2etest.grpc.io/v1
+kind: LoadTest
+metadata:
+  # Every load test instance must be assigned a unique name on the
+  # cluster. There are ways we can circumvent naming clashes, such
+  # as using namespaces or dynamically assigning names.
+  name: prebuilt-cxx-example
+
+  # As a custom resource, it behaves like a native kubernetes object.
+  # This means that users can perform CRUD operations through the
+  # Kubernetes API or kubectl. In addition, it means that the user
+  # can set any metadata on it.
+  labels:
+    language: cxx
+spec:
+  # The user can specify servers to use when running tests. The
+  # initial version only supports 1 server to limit scope. Servers
+  # is an array for future expansion.
+  #
+  # There are many designs and systems to pursue load balancing,
+  # organizing and monitoring a mesh of servers. Therefore, this
+  # will likely be expanded in the future.
+  servers:
+    - language: cxx
+      run:
+        image: ${prebuilt_image_prefix}/cxx:${prebuilt_image_tag}
```

This is a nit, but I don't like to include the trailing slash. If the image were, for example, an official image on dockerhub, it would contain no username. So it wouldn't be go/go. It would be just go. Including the slash makes it impossible to not provide a prefix, because /anything is not a valid image name in any repository afaik. 

```suggestion
        image: ${prebuilt_image_prefix}cxx:${prebuilt_image_tag}
```

---

I agree with that, but at this point some of Donghao's Pr has been merged already with this format, do you think it is okay to leave it as it is for now and when all the images are merged, I will remember to get all images updated?

---

That's okay with me. Yes, they should all be consistent, and if possible we should avoid the slash. Can be a follow-up.